app/api/discount_codes.py

```diff
@@ -288,3 +304,14 @@ def before_get(self, args, kwargs):
     schema = DiscountCodeSchemaEvent
     data_layer = {'session': db.session,
                   'model': DiscountCode}
+
+
+class DiscountCodeTicketRelationship(ResourceRelationship):
```

Comment:
Please check in other places. We don't need to define Relationship class for every relationship that we add.